content-usable/design.html

```diff
@@ -4442,8 +4443,8 @@ <h4>Technical Details</h4>
       <section>
         <h3>Pattern: Avoid Too Much Content</h3>
         <h4>User Need</h4>
-        <blockquote><p>???.</p></blockquote>
-        <p>Related User Story: <a href="#" aria-label="XX User Story">XXX</a>.</p>
+        <blockquote><p>I can easily identify content that I need, and do not need. Information I need to know and important information stands out, or is the first thing I read and does not get lost in the noise of less important information</p></blockquote>
```

Comment:
The User Need you have seems to better match other patterns, except for the last bit.